### ISSUE TITLE
Build problems

### ISSUE THREAD

**Issue by nathaniel-holder** [christianrondeau/jira-toggl-sync#12]
I'm having trouble building this project, specifically with the RestSharp assembly.  Is this a known issue?  What .Net Framework is supported, and what version of Visual Studio?  Sorry for newbie questions.

**Comment by christianrondeau** [comment]
No problem :) The solution should open with Visual Studio 2015 (though I've been running with 2017 for a while), and it's using .NET Framework 4.5.2.

What problem(s) do you have?

**Comment by nathaniel-holder** [comment]
Ahh, I figured it out: the project `TechTalk.JiraRestClient.csproj` had an incorrect `<HintPath>`.  Here's the diff:
```
diff --git a/TechTalk.JiraRestClient/TechTalk.JiraRestClient.csproj b/TechTalk.JiraRestClient/TechTalk.JiraRestClient.csproj
index e2b9e41..d289426 100644
--- a/TechTalk.JiraRestClient/TechTalk.JiraRestClient.csproj
+++ b/TechTalk.JiraRestClient/TechTalk.JiraRestClient.csproj
@@ -31,7 +31,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="RestSharp">
-      <HintPath>..\packages\RestSharp.104.1\lib\net4-client\RestSharp.dll</HintPath>
+      <HintPath>..\..\..\packages\RestSharp.104.1\lib\net4-client\RestSharp.dll</HintPath>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
```

**Comment by nathaniel-holder** [comment]
I'm getting Jira authorization denied.  Does this work with a cloud-based Jira account (eg with URL of `https://<myproject>.atlassian.net`)?  If so, does it work with two-step authentication?  Thanks so much!

**Comment by nathaniel-holder** [comment]
Ahh, figured it out again: I needed to use an [API token](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) instead of password when Two-step authentication is enabled.

**Comment by christianrondeau** [comment]
Weird that you had to change the `HintPath`. Is RestSharp in `\Modules\JiraRestClient\packages\RestSharp.104.1`? How did you do the NuGet package restore?

**Comment by christianrondeau** [comment]
Agreed @vipetrul. It would probably be simpler to just get rid of these submodules and simply call the json APIs directly.. But that's a pretty minor inconvenience. A note in the readme may be warranted.
